github/gh-collaboration.qmd

```diff
@@ -1,6 +1,12 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository click on the branch dropdown menue.

```

Comment:
```suggestion
- On the main page of the repository click on the branch dropdown menu.
```
Perhaps add the location of the drop-down menu on the page, i.e describe where it's placed.

I just realised that perhaps you're not referring to the drop-down menu but the button branches next to it. In which case you don't need 
```
- View all branches
```

Then then clears up what you're referring to in the sentence beneath the bullet points.

Reply:
I guess my point was that you  can either directly quickly select a branch OR go via "view all branches", meaning that there are more than one way to achieve the same thing,  but I see your point.

Reply:
Can you clarify this please in the text, as I didn't find the distinction clear.

Reply:
OK, updated to clarify this a bit better.